app/src/main/java/org/wikipedia/page/PageTitle.kt

```diff
@@ -49,7 +49,10 @@ data class PageTitle(
 
     var namespace: String
         get() = _namespace.orEmpty()
-        set(value) { _namespace = value; _displayText = null }
+        set(value) {
+            _namespace = value
+            _displayText = if (value.isEmpty()) _displayText else StringUtil.addUnderscores(value) + ":" + _displayText
```

Comment:
The `_displayText` should not contain underscores, since it's for display purposes. Should this be `removeUnderscores()` instead?

Reply:
Correct! Done!